source/geod24/concurrency.d

```diff
@@ -883,6 +886,31 @@ struct ThreadInfo
     }
 }
 
+
+/***************************************************************************
+
+    Getter of FiberScheduler assigned to a called thread.
+
+***************************************************************************/
+
+public @property FiberScheduler thisScheduler () nothrow
```

Comment:
Why do we need this ? Those global properties were really a problem in `std.concurrency` and we should try to remove them, not add more.

Reply:
I have a question.
How do we record and access additional information on the currently running thread?

Reply:
I made this new Thread class, and I solved it. What do you think about this?
```public class InfoThread : Thread
{
    public ThreadInfo info;
....
}
```